lib/code_scanning/rubocop/rule.rb

```diff
@@ -80,19 +104,19 @@ def to_h
         properties["description"] = desc
       end
 
-      unless help_empty?
-        help = @help.string
-        h["help"] = {
-          "text" => help,
-          "markdown" => help
-        }
-        properties["queryURI"] = query_uri if badge.qualified?
```

Comment:
Any particular reason you removed the 'queryURI' property?

Reply:
Yes, I had to remove it because the official SARIF validator fails the feed if queryURI is present. :/

Reply:
@indirect the code-scanning security tab and views rely on the `queryURI` to display some information. Also, we return it in our APIs. It seems like the SARIF validator does not recognize this field, but we have it in CodeQL and the rubocop generator too in order to provide the full experience. 

I am ok with adding the `helpUri`. but we can't remove the other, at least for now.

Reply:
Done, added queryURI back.